ONSdigital/eq-author-terraform
# Refactor WAF into its own terraform module.

### What is context of this PR?
Previously the WAF had been configured to only allow traffic from the UK.
This led to concourse failing to verify the status endpoint during deployment to staging because it is being hosted in IE.

This PR whitelists traffic originating from IE.

I have also moved the WAF configuration into its own terraform module, so that it may be deployed using concourse.

See https://github.com/ONSdigital/eq-author-terraform-waf.

### How to test?
Deploy an environment and ensure that the WAF is configured correctly to allow traffic from both GB and IE geo locations.
